source/main.cpp

```diff
@@ -293,7 +293,13 @@ void GameLoop(PlayerInfo &player, TaskQueue &queue, const Conversation &conversa
 				cursorTime = 0;
 
 			if(debugMode && event.type == SDL_KEYDOWN && event.key.keysym.sym == SDLK_BACKQUOTE)
+			{
 				isPaused = !isPaused;
+				if(isPaused)
+					Audio::Pause();
```

Comment:
Here it would make sense to pause also the music.

Reply:
Would it? It behaves pretty much the same way as anything else, as far as I can tell; the menu panels still get updated.

Reply:
I wouldn't mind having the music on when the game is paused
doesn't change anything until we actually get any musics anyway (or can we pause on a planet?)

Reply:
Yes, let's leave this for now as-is. (So let's just keep the music playing here, we can change it later.)